src/cmap/connection_pool.ts

```diff
@@ -323,6 +323,15 @@ export class ConnectionPool extends TypedEventEmitter<ConnectionPoolEvents> {
     return this[kServer].description.error;
   }
 
+  /**
+   * This is exposed ONLY for use in mongosh, so they can
```

Comment:
this is kind of nitpicky since this is currently removed from our generated API docs, but since these code comments are still in a way part of our public facing documentation, can we rephrase the sentence to use passive voice (e.g., "so they can kill connections" -> "to enable killing all connections") and also reference NODE-3263 (perhaps in a separate TODO comment)? Unless we are 100% confident that we aren't going to do NODE-3263 in any form, this getter might go away, or might be something that isn't used in the shell but is used by us instead to perform similar functionality on, say, `close(force = true)`